examples/run-ranger.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/python3
+
+# Opens ranger, if it's already opened focuses on it.
+# Usage:
+#   bindsym $mod+r  exec ~/.config/i3/Scripts/run_ranger.py $term
+
+import sys
+import i3ipc
+
+# Use xprop to find out the class of your terminal
+TERM_CLASS = 'URxvt'
```

Comment:
You can also `from ranger.ext.get_executables import get_term`, but this function does not work well with urxvt